src/SelfAssessment/AssessmentStart.spec.tsx

```diff
@@ -1,9 +1,15 @@
-import { render, fireEvent } from "@testing-library/react-native"
 import React from "react"
+
+import { render, fireEvent } from "@testing-library/react-native"
 import { I18nextProvider } from "react-i18next"
+import { useNavigation } from "@react-navigation/native"
 
 import i18n from "../locales/languages"
-import { AssessmentStart } from "./AssessmentStart"
+import AssessmentStart from "./AssessmentStart"
+
+const navigationSpy = jest.fn()
```

Comment:
How do you feel about moving this value closer to where it is used? I feel that it will be easier for readers to gain context on what is being tested if it happens closer to where is exercised.

Reply:
Yes good call!